packages/evaluatorq-py/src/evaluatorq.py

```diff
@@ -0,0 +1,117 @@
+from collections.abc import Awaitable, Sequence
+import asyncio
+import os
+import time
+
+from .table_display import display_results_table
+from .types import DataPoint, EvaluatorParams, EvaluatorqResult
+from src.processings import process_data_point
+from src.progress import ProgressService, with_progress, Phase
+
+
+def setup_orq_client(api_key: str):  # pyright: ignore[reportUnusedParameter]
+    pass
+
+
+def fetch_dataset_as_datapoints(orq_client, dataset_id: str):  # pyright: ignore[reportUnusedParameter,reportMissingParameterType,reportUnknownParameterType]
+    pass
+
+
+async def evaluatorq(name: str, params: EvaluatorParams) -> EvaluatorqResult:  # pyright: ignore[reportUnusedParameter]
```

Comment:
We typehint EvaluatorParams here, but don't actually validate this anywhere, right? I think we should add that, to make sure we are receiving the types and objects we expect.